### DISCUSSION TITLE
Natural gas

### DISCUSSION THREAD

**Discussion by ejg3855** [David00/rpi-power-monitor#60] [General]
Have you thought about incorporating a way to monitor natural gas flow meter into this project? All the ots solutions stink.

**Comment by David00** [comment]
I ejg3855, I have not.

**Comment by ejg3855** [comment]
I just ordered the bits and pieces from you to build this project, Once I have that ironed out I may look at it. Flow meters aren't cheap.

**Reply by David00** [reply]
Awesome, thanks! I'd be interested to hear your findings. I'd imagine that some of the cost comes from getting approval/certification to be installed inline with the existing gas system.

**Comment by beadon** [comment]
Regarding monitoring natural gas flow meters, since all of this is already monitored with an existing monitoring system for cost purposes is it worth developing a system that just reads the existing meter ?  I know many of these are bluetooth enabled, and baring that there are other contactless ways to get the information - eg:  visually inspecting the readout.  Thoughts ?

**Comment by ejg3855** [comment]
I was looking at a project that did visual comparison, mobile at the moment
so can't link it. But my meter is ancient haha

**Comment by beadon** [comment]
Ok, stupid question first then -- which model is it?  Is your gas provider willing to update it to a modern version?  They might just do it since it could save their agents time by using 'smart' meters.  Electric providers have taken this approach and it's saved the companies tons of $ and time.
